### PR TITLE
Introduce Avail Light Bootstrap releaser

### PR DESCRIPTION
**Pull request by obrienalaribe**

### CODE REVIEW

File: .github/workflows/avail-light-bootstrap.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      - '**'
+    tags:
+      - 'avail-light-bootstrap-v*.*.*'
+      - 'avail-light-bootstrap-v*.*.*-*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
```
Comment (moraesjeremias) [inline .github/workflows/avail-light-bootstrap.yml:14]: Do we consider including other OS in the future?

File: .github/workflows/avail-light-bootstrap.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    steps:
+      - name: Trigger repository_dispatch for release
+        shell: bash
+        run: |
+          curl -s -o /dev/null -w "%{http_code}" \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.PAT_TOKEN }}" \
+            https://api.github.com/repos/availproject/avail-light-infra/dispatches \
+            -d '{"event_type": "binary_release", "client_payload": {"bootnode_version": "${{ github.ref_name }}" }}'
```
Comment (moraesjeremias) [inline .github/workflows/avail-light-bootstrap.yml:78]: EOF

**Review by moraesjeremias — CHANGES_REQUESTED** [review]
What about relayer?

File: .github/workflows/avail-light-bootstrap.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+name: Bootstrap Release
+on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'avail-light-bootstrap-v*.*.*'
+      - 'avail-light-bootstrap-v*.*.*-*'
```
[the remark this answers is not in the document]
Comment (obrienalaribe) [reply .github/workflows/avail-light-bootstrap.yml:8]: Yup i think this pattern will remain the same for releases

[the remark this answers is not in the document]
Comment (moraesjeremias) [reply .github/workflows/avail-light-bootstrap.yml:8]: cool

File: .github/workflows/avail-light-client.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: Releaser
+name: Avail Light Release
```
Comment (momosh-ethernal) [inline .github/workflows/avail-light-client.yml:1]: Should we follow the full name of "Avail Light Client"?

**Review by momosh-ethernal — APPROVED** [review]

File: .github/workflows/avail-light-client.yml (hunk 4 above)
Comment (obrienalaribe) [reply .github/workflows/avail-light-client.yml]: Done

**Comment by momosh-ethernal** [comment]
> What about relayer?

We don't have to make releases for the Relay node, not yet.

**Review by momosh-ethernal — APPROVED** [review]

**Pull request merged**